bin/make-vps.sh

```diff
@@ -84,6 +84,11 @@ if [[ $? -ne 0 ]] ; then
   exit 1
 fi
 
+if [[ "${target_vlan:5:1}" == "3" && "${API_HOST:0:2}" == "g0" ]] || [[ "${target_vlan:5:1}" == "1" && "${API_HOST:0:2}" == "g1" ]] ; then
```

Comment:
This check should probably be part of `vlan_info()` in bin/vps-lib.sh.

I also think we should add an explicit cluster name to config, rather than rely on the API_HOST hostname.

Reply:
Ok, I've moved it to `vps-lib.sh` and set a variable for the cluster. I'm still using the API_HOST as I think that's a good anchor for what ganeti node we're talking to.